habitat-lab/habitat/tasks/rearrange/rearrange_sensors.py

```diff
@@ -543,7 +543,12 @@ def reset_metric(self, *args, **kwargs):
         self.update_metric(*args, **kwargs)
 
     def update_metric(self, *args, task, **kwargs):
-        if not task._is_episode_active and self._enable_gfx_replay_save:
+        is_timeout = False
+        if "is_timeout" in kwargs:
```

Comment:
This should always be passed I think? Just have it as a keyword argument in the method inputs.

Reply:
I think it does not get passed at init. I tried doing the keyword but was crashing